kobo/shortcuts.py

```diff
@@ -289,7 +290,7 @@ def run(cmd, show_cmd=False, stdout=False, logfile=None, can_fail=False, workdir
         if show_cmd:
             command = "COMMAND: %s\n%s\n" % (cmd, "-" * (len(cmd) + 9))
             if stdout:
-                print command,
+                print(command, end=' ')
```

Comment:
Should this be `end=''` instead? The printed string ends with a newline, so the space is printed at beginning of next line. Running `python -c 'import kobo.shortcuts; kobo.shortcuts.run("echo foo", show_cmd=True, stdout=True)'` shows different output than before migration (on any Python version).

This change is also included in https://github.com/frenzymadness/kobo/pull/2

Reply:
Yes, you are right, thank you!